programs/issue-2-temp-calculator/run.py

```diff
@@ -0,0 +1,39 @@
+def celsius_to_fahrenheit(celsius):
+    fahrenheit = celsius * 9/5 + 32
+    print("{}°C to {}°F".format(celsius, fahrenheit))
+
+
+def fahrenheit_to_celsius(fahrenheit):
+    celsius = (fahrenheit - 32) * 5/9
+    print("{}°F to {}°C".format(fahrenheit, celsius))
+
+
+def menu():
+    print("Program do konwersji temperatury w różnych skalach. Jaką konwersje chcesz wykonać?")
+    print("1) Stopnie Celsjusza na Fahrenheita")
+    print("2) Stopnie Fahrenheita na Celsjusza")
+    while True:
+        try:
+            user_request = int(input("Wybierz: "))
+        except ValueError:
+            print("Musisz podać numer opcji!")
+            continue
+        if not 2 >= user_request >= 1:
```

Comment:
dziwnie od tyłu się to czyta. Bardziej naturalnie było by:

`1 <= user_request <= 2:`

Reply:
Swoją drogą, python wyznaje zasadę:

> Easier to ask for forgiveness than permission. This common Python coding style assumes the existence of valid keys or attributes and catches exceptions if the assumption proves false. This clean and fast style is characterized by the presence of many try and except statements. The technique contrasts with the LBYL style common to many other languages such as C.

źródło: https://docs.python.org/3.4/glossary.html

Pomyśl, co byś musiał zrobić, byś mógł założyć, że użytkownik podad poprawną opcję, a jeżeli podałby złą, to poleciałby wyjątek rzucony przez pythona, a ty byś tylko go złapał i tam wyświetlił, nie istnieje taka opcja...

Reply:
> Pomyśl, co byś musiał zrobić
sam o tym wspomniałeś, więc dobrecyzuje tutaj... żebyś nad tym nie siedział teraz, bo to nie pora. Mógłbyś zrobic tak, by wszystkie dane do konwersji trzymać w jakimś słowniku czy liście... i jeżeli ktoś wybrał by coś z poza listy, to wtedy leciałby wyjątek...

Ale tego tutaj nie implementuj. Miałeś się tylko zastanowić :)